dspace-api/src/main/java/org/dspace/app/util/MetadataExposureServiceImpl.java

```diff
@@ -7,15 +7,14 @@
  */
 package org.dspace.app.util;
 
-import org.apache.log4j.Logger;
-import org.dspace.app.util.service.MetadataExposureService;
-import org.dspace.authorize.service.AuthorizeService;
-import org.dspace.core.ConfigurationManager;
-import org.dspace.core.Context;
-import org.springframework.beans.factory.annotation.Autowired;
-
-import java.sql.SQLException;
+import java.sql.*;
 import java.util.*;
+import org.apache.log4j.*;
+import org.dspace.app.util.service.*;
+import org.dspace.authorize.service.*;
+import org.dspace.core.*;
+import org.dspace.services.*;
+import org.springframework.beans.factory.annotation.*;
```

Comment:
Please note that per our new [Java Code Style](https://wiki.duraspace.org/display/DSPACE/Code+Style+Guide), wildcard import statements (i.e. those ending in asterisks) are no longer allowed (on the `master` branch).  

Since this PR is being submitted to the `dspace-6_x` branch, it need not follow the Code Style as strictly. However, in order to fix this issue on `master`, we'd need a second PR for `master` that does strictly follow the new Code Style.